### PR TITLE
Chuck/utf 8 ascii encoding

### PR DESCRIPTION
**Pull request by ccollins**
added dockerfile/compose
binstubs
pry

Force utf-8 encoding on text/html

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+PLATFORMS
+  aarch64-linux
+
+DEPENDENCIES
+  griddler!
+  griddler-sendgrid!
+  pry
+  rake
+  rspec (~> 3.12)
```
Comment (correlator) [inline Gemfile.lock:219]: Do we not need to remove griddler from the Gemfile?

File: bin/bundle [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        bundler_version = a
+      end
+      next unless a =~ /\A--bundler(?:[= ](#{Gem::Version::VERSION_PATTERN}))?\z/
+      bundler_version = $1
+      update_index = i
+    end
+    bundler_version
+  end
+
+  def gemfile
```
Comment (correlator) [inline bin/bundle:40]: where did this file come from?

**Review by correlator — COMMENTED** [review]
I'm kind of confused

File: bin/bundle (hunk 2 above)
Comment (ccollins) [reply bin/bundle:40]: binstubs

File: Gemfile.lock (hunk 1 above)
Comment (ccollins) [reply Gemfile.lock:219]: nope.  dependency for dev

**Comment by correlator** [comment]
Oh I get it now

**Pull request merged**